cspell.json

```diff
@@ -0,0 +1,62 @@
+{
+    "version": "0.1",
+    "language": "en",
+    "enabledLanguageIds": [
+        "css",
+        "graphql",
+        "html",
+        "json",
+        "javascript",
+        "javascriptreact",
+        "markdown",
+        "mdx",
+        "plaintext",
+        "restructuredtext",
+        "sass",
+        "scss",
+        "typescript",
+        "typescriptreact",
+        "yaml",
+        "yml"
+    ],
+    "ignorePaths": [
+        "cdai",
+        "coverage",
+        "css",
+        "es",
+        "lib",
+        "node_modules", // this will ignore anything the node_modules directory
+        "**/node_modules", // the same for this one
+        "**/node_modules/**", // the same for this one
```

Comment:
These are not required: the `node_modules` pattern matches node_modules appearing anywhere in the path, so these possibilities are already excluded.

```suggestion
```